backend/clients/twilio_client.py

```diff
@@ -175,6 +175,8 @@ async def send_whatsapp_message(self, body: str) -> None:
         except JSONDecodeError as e:
             logger.error(f"Error decoding JSON message: {e}")
         except TwilioRestException as e:
+            # TODO:: If message sent fail,
+            # how we want to handle resend message to twilio?
```

Comment:
Good question. I propose to just use a blocking loop for now. Just retry for an hour(?) and then log and drop. Not ideal, but we don't have the volume for this to be a problem. What do you think?

Reply:
Twilio has a maximum character limit of 1600, so I believe retrying isn't a viable option.
https://help.twilio.com/articles/360033806753-Maximum-Message-Length-with-Twilio-Programmable-Messaging

Reply:
Ah, sorry, I misunderstood. I thought this was for the message delivery, not the message content. In that case: there is little else to do than just (first) log an error and (sometime in the future) propagate the error back to the front-end user.